config/common/webhook/role-webhook.yaml

```diff
@@ -50,11 +50,3 @@ rules:
       - get
       - update
       - create
-  - apiGroups:
-      - apps
-    resources:
-      - daemonsets
-    verbs:
-      - get
-      - list
-      - watch
```

Comment:
this rule is missing and leading to errors in the webhook

Reply:
also `{Kind: "DeploymentConfig", APIVersion: "apps.openshift.io/v1"}` is missing

Reply:
Added